src/index.ts

```diff
@@ -657,6 +694,54 @@ async function mergeNeurons(sourceNeuronId: bigint, targetNeuronId: bigint) {
   ok();
 }
 
+async function registerVote(neuronId: bigint, proposalId: bigint, vote: Vote) {
+  if (!Object.values(Vote).includes(vote)) {
+    throw new Error(
+      `Invalid vote value. Valid values are: ${Object.values(Vote).join(", ")}`
+    );
+  }
+  const identity = await getIdentity();
+  const governance = GovernanceCanister.create({
+    agent: await getCurrentAgent(identity),
+  });
+
+  await governance.registerVote({
+    proposalId,
+    neuronId,
+    vote,
+  });
+
+  ok();
+}
+
+async function setFollowees(
+  neuronId: bigint,
+  topic: Topic,
+  followees: bigint[]
+) {
+  console.log(topic);
+  console.log(typeof topic);
+  if (!Object.values(Topic).includes(topic)) {
```

Comment:
this is alro repeated, worth a util?

Reply:
One is the `Topic` and the other is the `Vote` enum. I'm not sure it would help a lot to have an util to check whether a value is an enum.